help/de/Help_ch_Account-Actions.xml

```diff
@@ -1,910 +1,1049 @@
+        <para>Wo lässt sich das Konto in der Kontenhierarchie einordnen?
         </para>
       </listitem>
 
       <listitem>
-        <para>If there is an Opening Balance.
+        <para>Gibt es einen Anfangsbestand?
         </para>
       </listitem>
 
       <listitem>
-        <para>If there is a Commodity (security/currency) needed for the account.
+        <para>Soll das Konto Fremdwährungen oder Wertpapiere aufnehmen?
         </para>
       </listitem>
 
       <listitem>
-        <para>If online updating of the commodity price is needed.
+        <para>Wird eine Online-Aktualisierung des Währungs- oder Aktienkurses gewünscht?
         </para>
       </listitem>
     </itemizedlist>
 
-    <para>These details are described below.
+    <para>Im folgenden werden diese Einzelheiten beschrieben.
     </para>
 
     <sect2 id="gen-info">
-      <title>General Information Tab</title>
-
-      <para>The <guilabel>General</guilabel> tab is used to access the basic information about the account. It
-        provides a way of connecting the account to stock information if it is one of the currency,
-        mutual fund or stock account types. It can also be flagged as a
-        <link linkend="accts-placeholder">Placeholder</link> account. It displays if it is flagged
-        as a <guilabel>Tax Related</guilabel> account (which is set through the
+      <title>Allgemeine Information</title>
+
+      <para>Die Karteiseite <guilabel>Allgemein</guilabel> dient zur Pflege der grundlegenden Angaben zum Konto.
+        Hier steht auch die Möglichkeit zur Verfügung, das Konto mit einer Börsenkursquelle zu
+        verknüpfen falls es sich um ein Währungs-, Investmentfonds- oder Aktienkonto handelt.
+        Weiterhin kann es als ein <link linkend="accts-placeholder">Platzhalter</link>-Konto
+        markiert werden und zusätzlich wird angezeigt, wenn es als
+        <guilabel>Steuerrelevant</guilabel> gekennzeichnet ist (was in dem
         <menuchoice>
-          <guimenu>Edit</guimenu><guimenuitem>Tax Report Options</guimenuitem>
+          <guimenu>Bearbeiten</guimenu><guimenuitem>Optionen Steuerbericht</guimenuitem>
         </menuchoice>
-        dialog).
+        -Dialog erfolgt).
       </para>
 
-      <para>There are seven fields in the <guilabel>Identification</guilabel> section of this tab.
+      <para>Im oberen Abschnitt <guilabel>Identifizierung</guilabel> dieser Registerkarte stehen sieben
+        Eingabefelder zur Verfügung.
       </para>
 
-      <orderedlist>
+      <orderedlist spacing="compact">
         <listitem>
-          <para><guilabel>Account Name:</guilabel> The name for the account such as First Bank Checking.
+          <para><guilabel>Kontobezeichnung:</guilabel> Geben Sie einen Namen für das Konto ein, z.B. "Mein
+            Girokonto".
           </para>
         </listitem>
 
         <listitem>
-          <para><guilabel>Account Code:</guilabel> The optional number code as described in
```

Comment:
We removed the colons from the labels. For now 

- put it behind `</guilabel>` or 
- drop it and use `<varlist>…`